docs/examples/index.rst

```diff
@@ -23,6 +23,20 @@ Word Embedding
       Learn how to train fastText and word2vec embeddings on your own dataset, and determine
       embedding quality through intrinsic evaluation.
 
+   .. card::
+      :title: Extracting Sentence Features with Pre-trained ELMo
+      :link: sentence_embedding/elmo_sentence_representation.html
+
+      See how to use GluonNLP's model API to automatically download the pre-trained ELMo
+      model from NAACL2018 best paper, and extract features with it.
```

Comment:
See how to use GluonNLP's model API to automatically download the pre-trained ELMo
model from NAACL2018 best paper, and extract features with it.
-> 
See how to use GluonNLP's API to automatically download the pre-trained ELMo
model, and extract features from it.